chart/kubeapps/README.md

```diff
@@ -210,8 +223,69 @@ kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceac
 helm init --service-account tiller
 ```
 
-It is also possible, though less common, that your cluster does not have Role Based Access Control (RBAC) enabled. If this is the case you should perform the chart installation by setting `rbac.create=false`:
+It is also possible, though less common, that your cluster does not have Role Based Access Control (RBAC) enabled. To check if your cluster has RBAC you can execute:
+
+```console
+$ kubectl api-versions
+```
+
+If the above command does not include entries for `rbac.authorization.k8s.io` you should perform the chart installation by setting `rbac.create=false`:
```

Comment:
If this is the case, why do we need the `rbac.create` parameter? Couldn't we just conditionally create the RBAC resources if the API exists using the Helm capabilities object? Not saying we should change it, just wondering.

Reply:
Hmm what do you mean?

Reply:
if we have a condition:

```
{{- if (.Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1") -}}
```

and the version changes to `v2` we may not notice.

Reply:
Although in that case we might need to rewrite the RBAC rules anyway right?